metabrainz/templates/users/signup-commercial.html

```diff
@@ -61,10 +68,10 @@
 
     <div class="form-group">
       <label class="col-sm-4 control-label">
-        {{ form.description.label.text }}
+        {{ form.usage_desc.label.text }}
         <span style="color: red">*</span>
       </label>
-      <div class="col-sm-5">{{ form.description(class="form-control", required="required") }}</div>
+      <div class="col-sm-5">{{ form.usage_desc(class="form-control", required="required", maxlength=150) }}</div>
     </div>
```

Comment:
It would be nice to have a piece of text indicating the 150 characters max limit.